homeassistant/components/climate/const.py

```diff
@@ -1,19 +1,75 @@
 """Provides the constants needed for component."""
 
+# All activity disabled / Device is off/standby
+OPERATION_MODE_OFF = 'off'
+
+# Heating
+OPERATION_MODE_HEAT = 'heat'
+
+# Cooling
+OPERATION_MODE_COOL = 'cool'
+
+# The device supports heating/cooling to a range
+OPERATION_MODE_HEATCOOL = 'heatcool'
+
+# The temperature is set based on a schedule, learned behavior, AI or some
+# other related mechanism. User is not able to adjust the temperature
+OPERATION_MODE_AUTO = 'auto'
+
+# Device is in Dry/Huminity mode
+OPERATION_MODE_DRY = 'dry'
+
+# Only the fan is on, not fan and another mode like cool
+OPERATION_MODE_FAN_ONLY = 'fan_only'
+
+OPERATION_MODES = [
+    OPERATION_MODE_OFF,
+    OPERATION_MODE_HEAT,
+    OPERATION_MODE_COOL,
+    OPERATION_MODE_HEATCOOL,
+    OPERATION_MODE_AUTO,
+    OPERATION_MODE_DRY,
+    OPERATION_MODE_FAN_ONLY,
+]
+
+
+# Device is running an energy-saving mode
+HOLD_MODE_ECO = 'eco'
```

Comment:
So hold mode is what in some systems is called a profile?

Reply:
Yes and no. Profiles are only presetting of temperatures, but hold modes have their own life/circle or actions.

Reply:
But I'm fine to rename as profile

Reply:
I'm not proposing to rename it, just wanted to clarify.